Tools/TestWebKitAPI/Tests/WebKitCocoa/UnifiedPDFTests.mm

```diff
@@ -167,6 +169,20 @@
     Util::run(&done);
 }
 
+// FIXME: Combine this test with the WKWebView.IsDisplayingPDF test.
+UNIFIED_PDF_TEST(WKWebView_IsDisplayingPDF)
+{
+    auto webView = adoptNS([[TestWKWebView alloc] initWithFrame:CGRectMake(0, 0, 600, 600) configuration:configurationForWebViewTestingUnifiedPDF().get() addToWindow:YES]);
```

Comment:
Nit - I think we prefer explicit `RetainPtr` these days.

Reply:
Good point, fixed!